core/src/test/scala/org/apache/spark/sql/BaseTestGenerationSpec.scala

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2017 PingCAP, Inc.
+ * Copyright 2019 PingCAP, Inc.
```

Comment:
what's the rule about when to update the `Copyright year`?

Reply:
I forgot to use the right copyright year when I copy-pasted it from another file.